### ISSUE TITLE
[Feat] Hide the 'Console' tab

### ISSUE THREAD

**Issue by HuidaeCho** [OSGeo/grass#3210]
**Is your feature request related to a problem? Please describe.**

I use `g.gui` mainly as monitor + data catalog + layer management, and don't use the other tabs (Tools, Python, and Console). I want to hide all these three tabs, but we only have hide Tools and Python tabs in Preferences => General.

**Describe the solution you'd like**

I want to be able to hide the Console tab as well.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features
you've considered.

**Additional context**

**Comment by veroandreo** [comment]
What about the minimize and maximize buttons on that panel?
![image](https://github.com/OSGeo/grass/assets/20075188/480298b8-8b34-4bf0-89dd-77376e01442d)

it is possible to gain a lot of space just by minimizing:
![image](https://github.com/OSGeo/grass/assets/20075188/5605f997-0253-4d03-8bb7-c37bb681261d)

**Comment by HuidaeCho** [comment]
> What about the minimize and maximize buttons on that panel? ![image](https://user-images.githubusercontent.com/20075188/277170815-480298b8-8b34-4bf0-89dd-77376e01442d.png)
> 
> it is possible to gain a lot of space just by minimizing: ![image](https://user-images.githubusercontent.com/20075188/277170931-5605f997-0253-4d03-8bb7-c37bb681261d.png)

Yes, I do that as a workaround and I have to do that "every time" I start the GUI.
